pkg/inclusion/paths.go

```diff
@@ -32,8 +33,11 @@ func calculateCommitPaths(squareSize, start, msgShareLen int) []path {
 		if i == endRow {
 			end = normalizedEndIndex
 		}
-		coord := calculateSubTreeRootCoordinates(maxDepth, start, end)
-		for _, c := range coord {
+
+		subTreeRootMaxHeight := int(math.Log2(float64(types.MinSquareSize(msgShareLen))))
```

Comment:
Good point. Resolved in https://github.com/celestiaorg/celestia-app/pull/984/commits/670d84ee1371fb8b5cdcb7eb649152f398050cc9

Opted for a code comment b/c a function seemed like additional indirection (since it's only used in this one location) but could be convinced to extract a function if reviewers feel strongly that a function is preferable over a code comment